src/robot/utils/__init__.py

```diff
@@ -46,6 +46,7 @@
 from .markupwriters import HtmlWriter, XmlWriter, NullMarkupWriter
 from .importer import Importer
 from .islike import is_dict_like, is_list_like, is_str_like
+from .keyword import robot_name
```

Comment:
Been thinking this a little more:

- Decorators definitely need a separate module under `robot.api`.
- `robot.api.deco` ought to be a good enough name for the module. Other proposals can be presented, but I definitely don't want it be named `decorator` to avoid conflicts with [decorator module](https://pypi.python.org/pypi/decorator).